src/prommis/uky/uky_flowsheet.py

```diff
@@ -2005,35 +2033,34 @@ def function(stream):
                 m.fs.sl_sep2.liquid_inlet_state[0].flow_vol.unfix()
                 m.fs.sl_sep2.liquid_inlet_state[0].conc_mass_comp.unfix()
                 m.fs.sl_sep2.solid_state[0].flow_mol_comp.unfix()
-        elif stream == m.fs.precip_sep:
-            print(f"Initializing {stream}")
-            initializer_sep.initialize(m.fs.precip_sep)
-        elif stream == m.fs.precip_sx_mixer:
-            print(f"Initializing {stream}")
-            initializer_mix.initialize(m.fs.precip_sx_mixer)
         else:
-            print(f"Initializing {stream}")
-            initializer_bt.initialize(stream)
+            print(f"{unit} is not being initialized.")
```

Comment:
What units are not being initialized, and why?

Also, this is still a lot of `if/else` statements. Is there a more efficient way you could do this? Also, seeing as the BT Initializer is more or less the default, would it be possible to have it be the blanket `else`?

Reply:
All of the units were being initialized. This was just a leftover check that I forgot to delete when making sure all units were initialized. I've made the BT Initializer the blank else statement now, but I'm not sure if there's a more efficient way to do this. 

The main issue is that the SolventExtraction models don't work with the BTInitializer or the MSContactorInitializer. Is the following error something wrong with the MSContactor model, the SolventExtraction model, or my flowsheet?
![image](https://github.com/prommis/prommis/assets/96305519/5bb159f8-18f1-4c0f-9264-295a3f3fd2ad)

Reply:
To my knowledge, the only DOFs I need to fix for the SX models in `set_operating_conditions` are the partition coefficients, so I'm not even sure what these 36 DOFs are.